front/app/containers/Admin/projects/project/offlineInputs/TopBar/index.tsx

```diff
@@ -24,12 +25,24 @@ interface Props {
 
 const TopBar = ({ onClickPDFImport }: Props) => {
   const localize = useLocalize();
-  const { projectId } = useParams() as {
+  const { projectId, phaseId } = useParams() as {
     projectId: string;
+    phaseId: string;
   };
 
   const { data: project } = useProjectById(projectId);
-  const projectTitle = project?.data.attributes.title_multiloc;
+  const { data: phase } = usePhase(phaseId);
+
+  const topBarTitle =
+    localize(project?.data.attributes.title_multiloc) +
+    (phase ? ` - ${localize(phase?.data.attributes.title_multiloc)}` : '');
+
+  const isSurvey =
+    phase || project?.data.attributes.participation_method === 'native_survey';
```

Comment:
```suggestion
    phase?.data.attributes.participation_method === 'native_survey' || project?.data.attributes.participation_method === 'native_survey';
```

I understand that `phaseId` only happens to be in the parameters when we are dealing with a survey, but that is context that you don't have when you are just looking at this file in isolation. This is functionally the same but less confusing for someone who is not aware of this context